### PR TITLE
Switch to webm for videos

### PR DESCRIPTION
**Pull request by hvanyo**

### CODE REVIEW

**Review by wodenx — DISMISSED** [review]

File: packages/bodiless-demo/src/components/Video/VideoClean.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    // prop and pass it as a ref to the underlying html element.
+    // @see https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/fclasses/src/addClasses.tsx
+    <C.Wrapper forwardRef={containerRef}>
+      <C.Video forwardRef={videoRef} muted="muted">
+        Sorry, your browser does not support videos
+      </C.Video>
+    </C.Wrapper>
+  );
+
+  return (
```
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/VideoClean.tsx]: @wodenx  two returns, delete this one.

File: packages/bodiless-demo/src/components/Video/AutoPlayback.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  // catching the unmounted error
+  if (!containerRef || !videoRef) return [null, null];
+
+  // @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+  const cb = entries => {
+    const [entry] = entries;
+    if (videoRef.current) {
+      if (entry.isIntersecting) {
+        // https://developer.chrome.com/blog/play-request-was-interrupted/
+        const playPromise = videoRef.current!.play();
```
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/AutoPlayback.tsx:19]: Error in function IntersectionObserver.cb in [http://localhost:](http://localhost/)
Cannot read properties of null (reading 'pause')  

--  handle this with the ! non-null operator on both on this line and line 30

File: packages/bodiless-demo/src/components/Video/AutoPlayback.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { useRef, useEffect } from 'react';
+
+// @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+const useVideoAutoPlayback = options => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  // catching the unmounted error
+  if (!containerRef || !videoRef) return [null, null];
```
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/AutoPlayback.tsx:11]: Issue:  in edit interface -- if I hit reload or swapped between edit/preview, it sometimes hit unmounted error

Added this check if I both valid refs and then passed back null one of them wasn't valid.

File: packages/bodiless-demo/src/components/Video/AutoPlayback.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          playPromise.then(_ => {
+            // Automatic playback started!
+            // Show playing UI.
+          })
+            .catch(error => {
+            // Auto-play was prevented
+            // Show paused UI.
+            });
+        }
+      } else videoRef.current!.pause();
```
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/AutoPlayback.tsx:30]: Issue: Error in function IntersectionObserver.cb in [http://localhost:](http://localhost/)
Cannot read properties of null (reading 'pause')  

--  handle this with the ! non-null operator on both on this line and line 19

File: packages/bodiless-demo/src/components/Video/AutoPlayback.tsx (hunk 2 above)
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/AutoPlayback.tsx:19]: Issue: in edit interface -- if I clicked add component then it threw a dom exception The play() request was interrupted by a call to pause().

add the try/check on play at lines 18 - 30 per doc

File: packages/bodiless-demo/src/components/Video/AutoPlayback.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
+// from https://esausilva.com/2021/06/14/react-hook-to-play-video-using-intersection-observer/
+
+import { useRef, useEffect } from 'react';
+
+// @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+const useVideoAutoPlayback = options => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  // catching the unmounted error
```
Comment (hvanyo) [inline packages/bodiless-demo/src/components/Video/AutoPlayback.tsx]:
```suggestion
  // Catching the unmounted error
```

**Review by wodenx — DISMISSED** [review]

**Pull request merged**